ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml

```diff
@@ -133,7 +153,7 @@ tests:
       ASSISTED_CONFIG: |
         TEST_FUNC=test_olm_operator
       TEST_SUITE: full
-    workflow: baremetalds-assisted
+    workflow: baremetalds-assisted-install
```

Comment:
you can remove all those changes, no need for you to see failures related to them

Reply:
done